### ISSUE TITLE
[Bug]: Wrong token used in tag input for disabled status

### ISSUE THREAD

**Issue by nouha06** [Orange-OpenSource/ouds-flutter#310]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Samsung A22

### OS Version

Android 13

### App version

v0.5.0

### Describe the issue

The background color of the tag input is wrong when status is 'Disabled'

<img width="200" height="800" alt="Image" src="https://github.com/user-attachments/assets/b49ed17a-5e7b-4594-a3c0-77393cefdf8d" />

### Expected Behavior

The background color for the tag input when status is Disabled should be transparent

<img width="200" height="800" alt="Image" src="https://github.com/user-attachments/assets/ac86a521-b0f6-44a0-beb8-2a13e3e92ec4" />

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
